tests/kuttl/tests/keystone_scale/01-assert.yaml

```diff
@@ -0,0 +1,62 @@
+#
+# Check for:
+#
+# - 1 MariaDB CR
+# - 1 Pod for MariaDB CR
+#
+
+apiVersion: mariadb.openstack.org/v1beta1
+kind: MariaDB
+metadata:
+  name: openstack
+  namespace: openstack
+spec:
+  containerImage: quay.io/tripleozedcentos9/openstack-mariadb:current-tripleo
+  secret: osp-secret
+  storageClass: local-storage
+  storageRequest: 500M
+status:
+  conditions:
+  - message: Setup complete
+    reason: Ready
+    status: "True"
+    type: Ready
+  - message: Deployment completed
+    reason: Ready
+    status: "True"
+    type: DeploymentReady
+  - message: Exposing service completed
+    reason: Ready
+    status: "True"
+    type: ExposeServiceReady
+  - message: MariaDB dbinit completed
+    reason: Ready
+    status: "True"
+    type: MariaDBInitialized
+  - message: Service config create completed
+    reason: Ready
+    status: "True"
+    type: ServiceConfigReady
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    openshift.io/scc: anyuid
+  labels:
+    app: mariadb
+    cr: mariadb-openstack
+    owner: mariadb-operator
+  name: mariadb-openstack
+  namespace: openstack
+spec:
+  containers:
+  - image: quay.io/tripleozedcentos9/openstack-mariadb:current-tripleo
+    imagePullPolicy: IfNotPresent
+    name: mariadb
+    resources: {}
+  restartPolicy: Always
+  serviceAccount: mariadb-operator-mariadb
+  serviceAccountName: mariadb-operator-mariadb
+status:
+  phase: Running
```

Comment:
if possible this should be in the mariadb-operator repo and can be validated by some `make mariadb_deploy_validate` or something. like with `make mariadb_deploy` it could then be consumed by any other service operator and if there is a change we won't have to update all asserts in all the operators.

We should have this for all the operators for the basic setup make <service>_deploy. with this installing
the dependencies for an operator and validate that the service is ok would be easy to implement and maintained.

Reply:
or have a make target which copies this base validation assert file from the mariadb-operator here where it is needed to be checked...

Reply:
I agree.  For common dependency operators like MariaDB and Keystone, we need to have some way of supporting a centralized definition of "success" for their deployment.  i.e. when I was the `install_yamls` KUTTL example, I did something like this: https://github.com/abays/install_yamls/blob/kuttl/tests/kuttl/common/tests/assert/deploy-keystone.yaml.  The question in this distributed context becomes how/where do we store/represent these common objects?